monitoring/dashboard.json

```diff
@@ -0,0 +1,627 @@
+    "panels": [
+        {
+            "collapsed": false,
+            "datasource": null,
+            "gridPos": {
+                "h": 1,
+                "w": 24,
+                "x": 0,
+                "y": 0
+            },
+            "id": 2,
+            "panels": [],
+            "title": "Response codes",
+            "type": "row"
+        },
+        {
+            "datasource": null,
+            "fieldConfig": {
+                "defaults": {
+                    "color": {
+                        "mode": "palette-classic"
+                    },
+                    "custom": {
+                        "axisLabel": "",
+                        "axisPlacement": "auto",
+                        "barAlignment": 0,
+                        "drawStyle": "line",
+                        "fillOpacity": 0,
+                        "gradientMode": "none",
+                        "hideFrom": {
+                            "legend": false,
+                            "tooltip": false,
+                            "viz": false
+                        },
+                        "lineInterpolation": "linear",
+                        "lineWidth": 1,
+                        "pointSize": 5,
+                        "scaleDistribution": {
+                            "type": "linear"
+                        },
+                        "showPoints": "auto",
+                        "spanNulls": false,
+                        "stacking": {
+                            "group": "A",
+                            "mode": "none"
+                        },
+                        "thresholdsStyle": {
+                            "mode": "off"
+                        }
+                    },
+                    "mappings": [],
+                    "thresholds": {
+                        "mode": "absolute",
+                        "steps": [
+                            {
+                                "color": "green",
+                                "value": null
+                            },
+                            {
+                                "color": "red",
+                                "value": 80
+                            }
+                        ]
+                    }
+                },
+                "overrides": []
+            },
+            "gridPos": {
+                "h": 8,
+                "w": 10,
+                "x": 0,
+                "y": 1
+            },
+            "id": 4,
+            "options": {
+                "legend": {
+                    "calcs": [],
+                    "displayMode": "list",
+                    "placement": "bottom"
+                },
+                "tooltip": {
+                    "mode": "single"
+                }
+            },
+            "targets": [
+                {
+                    "exemplar": true,
+                    "expr": "sum by (code) (rate(http_requests_total{namespace=\"${namespace}\", job=\"${job}\"}[10m]))",
```

Comment:
should this rate here be computed over a fixed 10m window (or smaller window? i used 1 min for alerts, but i don't know what value is appropriate, as we discussed on alerts PR...), or use the window that is specified in grafana?

(the answer may be more obvious with the dashboard in front, but looking at the JSON it is not very clear to me, sorry...)

Reply:
in fact 10 minutes is quick short, previously in xcore for a graph over 7 days we used a 5 hours window, it give the tendency of the (bandwidth|latency|....)

here I used 10 minutes but later with some real workload will change it to a longer time range. 

we don´t want to use the grafana value for the window the user is looking at because with a window of 7 days for the graph we want something like 1/10th of it for the rate window or something similar.

Reply:
looking at the doc, we should thus use `$__rate_interval` to handle all cases: https://grafana.com/blog/2020/09/28/new-in-grafana-7.2-__rate_interval-for-prometheus-rate-queries-that-just-work/

Reply:
looking at the article now, thanks did not know about that.